ci/kokoro/windows/build-cmake.ps1

```diff
@@ -88,7 +88,12 @@ if (Test-Path env:RUNNING_CI) {
 # Get the number of processors to parallelize the tests
 $NCPU=(Get-CimInstance Win32_ComputerSystem).NumberOfLogicalProcessors
 
-$ctest_flags = @("--output-on-failure", "-j", $NCPU, "-C", $env:CONFIG)
+$ctest_flags = @(
```

Comment:
`s/ctest_flags/ctest_args/` to match?

Reply:
Done, and also s/cmake_flags/cmake_args/ so the file is internally consistent.